sympy/utilities/lambdify.py

```diff
@@ -515,36 +522,51 @@ def lambdastr(args, expr, printer=None, dummify=False):
         from sympy.printing.lambdarepr import lambdarepr
 
     def sub_args(args, dummies_dict):
+       
         if isinstance(args, str):
-            return args
+           return args
         elif isinstance(args, DeferredVector):
-            return str(args)
+           return str(args)
         elif iterable(args):
             dummies = flatten([sub_args(a, dummies_dict) for a in args])
-            return ",".join(str(a) for a in dummies)
+            return ",".join(str(a) for a in dummies if a!=None)
+
+        
         else:
             #Sub in dummy variables for functions or symbols
             if isinstance(args, (Function, Symbol)):
+                
                 dummies = Dummy()
                 dummies_dict.update({args : dummies})
                 return str(dummies)
+
+            elif isinstance(args,Derivative):
+                dummies = Dummy()
+                dummies_dict.update({args : dummies})
```

Comment:
I see that the whitespace before the colon was there already, but it shouldn't be. According to PEP8, it should be `{args: dummies}`.